### PR TITLE
[SIG-2910] Added some small scenario's

### PR DESCRIPTION
**Pull request by arjandata**

### CODE REVIEW

File: cypress/integration/changeSignalAfterSubmit.spec.js [hunk 1]
```diff
@@ -132,6 +132,9 @@ describe('Change signal after submit', () => {
       cy.wait(1000);
       cy.get(CHANGE_LOCATION.buttonSubmit).click();
 
+      // Check if background colour of changed element is flashing orange
+      createSignal.checkFlashingOrange();
```
Comment (janjaap) [inline cypress/integration/changeSignalAfterSubmit.spec.js:136]: It's actually yellow

File: cypress/support/commandsCreateSignal.js [hunk 2]
```diff
@@ -24,6 +25,28 @@ export const checkDescriptionPage = () => {
...
   cy.contains('Voeg een foto toe om de situatie te verduidelijken').should('be.visible');
+  cy.get('body').then($body => {
+    if ($body.find(`${CREATE_SIGNAL.disclaimer}`).length > 0) {
+      checkHeaderFooter();
+    }
+  });
+};
+
+export const checkFlashingOrange = () => {
+  cy.get('.highlight--active')
```
Comment (janjaap) [inline cypress/support/commandsCreateSignal.js:36]: There's a pending change that will no longer have this class name available on the element. The element, however, will be targetable by the attribute `data-testid="highlight"` and, when active, with class name `animate`.

File: cypress/support/selectorsSignalDetails.js [hunk 3]
```diff
@@ -17,7 +18,7 @@ export const CHANGE_STATUS = {
   buttonClose: '[class*="incident-detail__preview-close"]',
```
Comment (janjaap) [inline cypress/support/selectorsSignalDetails.js:18]: Close buttons will be targetable by `data-testid="closeButton"`

File: cypress/integration/changeSignalAfterSubmit.spec.js (hunk 1 above)
Comment (arjandata) [reply cypress/integration/changeSignalAfterSubmit.spec.js:136]: Wow, difficult to see :-) But you are right.

File: cypress/support/selectorsSignalDetails.js (hunk 3 above)
Comment (arjandata) [reply cypress/support/selectorsSignalDetails.js:18]: There is no `data-testid="closeButton"` for the X button when changing status

Comment (janjaap) [reply cypress/support/selectorsSignalDetails.js:18]: 'will be' Targetable. Not right now, but they will after #712 has been merged. It was more like a heads-up.

**Pull request merged**